cli/cmd/compliance_aws.go

```diff
@@ -236,6 +250,10 @@ func init() {
 		"download report in PDF format",
 	)
 
+	complianceAwsGetReportCmd.Flags().BoolVar(&compCmdState.Csv, "csv", false,
+		"download report in CSV format",
```

Comment:
if we are not downloading a report to disk, let's just say we change the format/output